rrfs-test/CMakeLists.txt

```diff
@@ -80,12 +80,23 @@ if(MPAS_DYCORE)
       file(COPY ${src_casedir}/namelist.atmosphere_15km  DESTINATION ${casedir})
 
    endforeach()
-
-   set(target_test rrfs_mpasjedi_2022052619_Ens3Dvar)
-   ecbuild_add_test( TARGET   ${target_test} 
+  message(STATUS "MACHINCE_ID is " ${MACHINE_ID})
+   if("${MACHINE_ID}" STREQUAL "orion")
```

Comment:
```suggestion
   if("$ENV{MACHINE_ID}" STREQUAL "orion")
```

Reply:
Good.  But, then, we need to export it in the build.sh , right?  otherwise, env in the child script will not get it by env. 
I will try it.

Reply:
probably but not 100% sure